### PR TITLE
Add GitSense extension

### PR DESCRIPTION
**Pull request by gitsense**
Hi,

I would like to update the README file to include the GitSense Chrome extension.

Thanks

### CODE REVIEW

**Comment by laughedelic** [comment]
It seems that this extension is available in the chrome app store. So what is this beta-participating email about?

**Comment by gitsense** [comment]
@laughedelic GitSense is composed of a client interface and an indexing engine.  The client interface is currently available in the Chrome store and the indexing engine is currently being managed by us.

We are currently indexing about 5000 popular GitHub repos and this is pretty much its limit.  It's not that it can't index more, it's just that GitHub could shut us down pretty quickly for abusing their resources if we started to index 10s of thousands of repos.

What the beta trial is for, is to make it possible for GitHubbers to manage their own indexing engine.  We are currently working on a list of host providers that will let GitHub/Bitbucket users install our indexing engine in the cloud for free.  By managing their own indexing engine, users will be able index whatever public/private GitHub repos they want.  

Instead of send an email to privatebeta@gitsense.com in the README, it would probably make sense to just provide a link to a web page where we can explain how the beta trial works in greater detail.

**Comment by stefanbuck** [comment]
Looks good, but could you remove the comparison table, please. It doesn't fit with the style of list. Instead of I would like to see an extended description of what the extension is for. Finally, to avoid any further confusion, can you rephrase the beta participating sentence, please. Thanks

**Comment by gitsense** [comment]
@stefanbuck I agree with the suggest changes.  I'll try to submit another update later today (Pacific time) or tomorrow.

Cheers

**Comment by stefanbuck** [comment]
Thanks :+1:

**Comment by gitsense** [comment]
@stefanbuck 

I'm going to close this pull request and I'll create a new one that has a cleaner history when I get the time.

Cheers
